docker-compose.yml

```diff
@@ -138,3 +138,13 @@ services:
       - JAVA_OPTIONS=-Dorg.eclipse.jetty.annotations.AnnotationParser.LEVEL=OFF
       - XMS=256M
       - XMX=1G
+
+  mapstore:
+    image: geosolutionsit/mapstore2-georchestra:latest
```

Comment:
Would be perfect if https://github.com/georchestra/mapstore2-georchestra/blob/master/.github/workflows/mapstore.yml could generate a `georchestra/mapstore` image, as for the other apps.

Reply:
yup, appending something like below in https://github.com/georchestra/mapstore2-georchestra/blob/master/.github/workflows/mapstore.yml should do the trick, but only then we can replace the docker hub image in the compose: 
```
    - name: "Building docker image"
      if: github.repository == 'georchestra/georchestra'
      working-directory: mapstore2-georchestra/
      run: ../mvnw --no-transfer-progress clean package docker:build -Pdocker,log4j-logstash,sentry-log4j  -DdockerImageName=georchestra/mapstore:${{ steps.version.outputs.VERSION }} -DskipTests

    - name: "Logging in docker.io"
      if: github.repository == 'georchestra/georchestra'
      uses: azure/docker-login@v1
      with:
        username: '${{ secrets.DOCKER_HUB_USERNAME }}'
        password: '${{ secrets.DOCKER_HUB_PASSWORD }}'

    - name: "Pushing latest to docker.io"
      if: github.ref == 'refs/heads/master' && github.repository == 'georchestra/georchestra'
      run: |
        docker tag georchestra/mapstore:${{ steps.version.outputs.VERSION }} georchestra/mapstore:latest
        docker push georchestra/mapstore:latest
    - name: "Pushing release branch to docker.io"
      if: contains(github.ref, 'refs/heads/20.') && github.repository == 'georchestra/georchestra'
      run: |
        docker push georchestra/mapstore:${{ steps.version.outputs.VERSION }}
    - name: "Pushing release tag to docker.io"
      if: contains(github.ref, 'refs/tags/20.') && github.repository == 'georchestra/georchestra'
      run: |
        docker push georchestra/mapstore:${{ steps.version.outputs.VERSION }}
```

Reply:
Please PR ;-)

Reply:
@fvanderbiest I got up to here for workflow docker credentials need to be configured:
 https://github.com/georchestra/mapstore2-georchestra/runs/1158835723?check_suite_focus=true

Reply:
Please create the PR and we'll review it.